src/components/schedule-page/table/schedule/schedule.component.tsx

```diff
@@ -17,37 +19,119 @@ export function ScheduleComponent(): JSX.Element {
     setNewSchedule(scheduleModel);
   }, [scheduleModel, setNewSchedule]);
 
+  function isPresent(): boolean {
+    if (scheduleLocalState.isInitialized) {
+      const a = scheduleLocalState.dateSection?.length;
+      const b = scheduleLocalState.nurseShiftsSection?.length;
+      const c = scheduleLocalState.babysitterShiftsSection?.length;
+
+      if (a && b && b && c && a * b * c) {
+        return true;
+      }
+    }
+    return false;
+  }
+
   return (
     <React.Fragment>
-      {scheduleLocalState.isInitialized && (
-        <table className="table">
+      {!isPresent() && (
+        <table>
+          <tbody>
+            <tr>
+              <td></td>
+              <td>
+                <div className="c" style={{ marginLeft: 126, marginTop: 20 }}>
+                  <TimeTableComponent />
+                </div>
+              </td>
+              <td></td>
+            </tr>
+          </tbody>
+        </table>
+      )}
+      {isPresent() && (
+        <table style={{ margin: 20 }}>
           <tbody>
             <ScheduleLogicContext.Provider value={scheduleLogic}>
-              <DateSectionComponent
-                uuid={scheduleLocalState.uuid}
-                data={scheduleLocalState.dateSection}
-              />
-
-              <EmptyRowComponent />
-
-              <FoundationInfoComponent
-                uuid={scheduleLocalState.uuid}
-                data={scheduleLocalState.foundationInfoSection}
-              />
-
-              <ShiftsSectionComponent
-                uuid={scheduleLocalState.uuid}
-                workerType={WorkerType.NURSE}
-                data={scheduleLocalState.nurseShiftsSection}
-              />
-
-              <EmptyRowComponent />
-
-              <ShiftsSectionComponent
-                uuid={scheduleLocalState.uuid}
-                workerType={WorkerType.OTHER}
-                data={scheduleLocalState.babysitterShiftsSection}
-              />
+              <tr className="sectionContainer">
```

Comment:
W sumie jak teraz myślę, to z kolumn <tr> można zrobić komponenty

Reply:
Nie jestem pewien co do tego czy  to jest potrzebne:
My możemy spokojnie manipulować wybieraniem kolumn z poziomu tego obiektu grafiku co my mamy.
Chyba, że ty widzisz jakieś zalety które mi umknęły?